app/src/main/res/layout/activity_bill.xml

```diff
@@ -0,0 +1,287 @@
+                                android:hint="@string/title_customer_name"
+                                android:inputType="textPersonName"
+                                android:text="@={viewModel.billRequest.customerName}"
+                                android:textSize="@dimen/sp_12"/>
+                        </android.support.design.widget.TextInputLayout>
+                    </LinearLayout>
+
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_10">
+
+                        <ImageView
+                            tools:ignore="ContentDescription"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center_vertical"
+                            app:srcCompat="@drawable/ic_phone_receiver_24dp"/>
+
+                        <android.support.design.widget.TextInputLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8">
+
+                            <EditText
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:hint="@string/title_customer_phone"
+                                android:inputType="textPersonName"
+                                android:text="@={viewModel.billRequest.phone}"
+                                android:textSize="@dimen/sp_12"/>
+                        </android.support.design.widget.TextInputLayout>
+                    </LinearLayout>
+                </LinearLayout>
+            </android.support.v7.widget.CardView>
+
+            <android.support.v7.widget.CardView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/dp_5"
+                bind:cardBackgroundColor="@color/color_white"
+                car_view:cardElevation="@dimen/dp1"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:baselineAligned="false"
+                    android:orientation="horizontal">
+
+                    <LinearLayout
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
+                        android:orientation="vertical"
+                        android:paddingStart="@dimen/dp_10">
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 15) you should *also* add `android:paddingLeft="@dimen/dp_10"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.